pkg/minikube/bootstrapper/images/images.go

```diff
@@ -64,24 +77,65 @@ func componentImage(name string, v semver.Version, mirror string) string {
 	return fmt.Sprintf("%s:v%s", path.Join(kubernetesRepo(mirror), name), v)
 }
 
+// fixes 13136 by getting the latest image version from the k8s.gcr.io repository instead of hardcoded
+func findLatestTagFromRepository(url string, lastKnownGood string) string {
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", url, nil)
+	errorMsg := fmt.Sprintf("Failed to get latest image version for %s, reverting to version %s.", url, lastKnownGood)
+	if err != nil {
+		klog.Warningf("%s Error %v", errorMsg, err)
+		return lastKnownGood
+	}
+
+	resp, err := client.Do(req)
```

Comment:
This can be changed to
```
resp, err := client.Get(url)
```
Which will remove these lines
```
	client := &http.Client{}
	req, err := http.NewRequest("GET", url, nil)
	if err != nil {
		klog.Warningf("%s Error %v", errorMsg, err)
		return lastKnownGood
	}
```

Reply:
Thanks! Changed.